src/app/state/staking/types.ts

```diff
@@ -39,6 +39,7 @@ export interface ValidatorMediaInfo {
   email_address?: string
   twitter_acc?: string
   tg_chat?: string
+  logotype?: string
```

Comment:
huh, shouldn't this have been generated from the swagger thing? or is this another type

Reply:
No, that's a wrapper on top of swagger's structures.